addons/website/static/src/js/menu/edit.js

```diff
@@ -51,7 +51,7 @@ var EditPageMenu = websiteNavbarData.WebsiteNavbarActionWidget.extend({
         });
         this.oeStructureSelector = '#wrapwrap .oe_structure[data-oe-xpath][data-oe-id]';
         this.oeFieldSelector = '#wrapwrap [data-oe-field]';
-        this.oeCoverSelector = '#wrapwrap [data-res-model][data-name="Cover"]';
+        this.oeCoverSelector = '#wrapwrap .s_cover[data-res-model], #wrapwrap .o_record_cover_container[data-res-model]';
```

Comment:
Are you sure this is the right fix? How come I sometimes manage to save and sometimes I don't ? If your commit explanation is right, I should never be able to, right?

Reply:
You can save it if you also impact sub-elements among your changes. The sub-element became dirty and therefore the whole thing got saved - including the non-dirty-marked change on `.o_record_cover_container`.
E.g.: the background color is on the root element, the background image and the filter intensity are on inner divs.